### ISSUE TITLE
Could not find TrajectryTracker form simulator package

### ISSUE THREAD

**Issue by Sudar278** [yifanlu0227/ChatSim#18]
![IMG-20240429-WA0007](https://github.com/yifanlu0227/ChatSim/assets/119469584/d00b7d6b-04e2-4bbb-bedf-feedde140fa4)

Got a error while the motion agent is activated
The error is a module error. Simulator package has no module TrajectoryTracker

In the below image I have printed all the modules from the library but I could not find TrajectoryTracking in that

![IMG-20240429-WA0008](https://github.com/yifanlu0227/ChatSim/assets/119469584/39272140-162d-4883-a123-da9ea1e40221)

Hope you guys have a solution for this 

Thanks in advance

**Comment by yifanlu0227** [comment]
<img width="1108" alt="Screenshot 2024-04-30 at 09 21 32" src="https://github.com/yifanlu0227/ChatSim/assets/45688237/c72e50a8-84d1-4637-8a50-7f95ffc0567f">
 Oh I think you need to install this lib (we updated the README some time ago). Right?@vfishc

**Comment by vfishc** [comment]
Yes, you can install the trajectory tracking module for smoother and more realistic trajectories, and you can also set the motion_agent-motion_tracking as False in config to disable this part.

**Comment by NiveVijay** [comment]
When motion_agent-motion_tracking is set as false in config, 
We could successfully run it, but the generated video is not playing, and all the images rendered look the same.

Below are the attached files. Please have a look.
[1137_demo_2024_04_30_10_27_32.zip](https://github.com/yifanlu0227/ChatSim/files/15159578/1137_demo_2024_04_30_10_27_32.zip)

Due to space constraints in uploading, the images in the folder are manually deleted, except 2. The deleted images look similar to the images provided in the above folder.

**Comment by NiveVijay** [comment]
In the method of cloning the git repository given in step 6, after cloning the given repository, I am getting the following error mentioned in the image below:
ModuleNotFoundError: No module named 'drltt_proto'

![image](https://github.com/yifanlu0227/ChatSim/assets/126130233/17b2cb8a-8283-45f9-a133-9be41c57315f)

**Comment by mahdibkh1996** [comment]
Try running these:

export PYTHONPATH=$PYTHONPATH:/home/<username>/ChatSim/chatsim/foreground/drl-based-trajectory-tracking/common/proto/proto_gen_py
export PYTHONPATH=$PYTHONPATH:/home/<username>/ChatSim/chatsim/foreground/drl-based-trajectory-tracking/sdk/proto_gen
export PYTHONPATH=$PYTHONPATH:/home/<username>/ChatSim/chatsim/foreground/drl-based-trajectory-tracking
